generators/client/templates/react/src/main/webapp/app/modules/administration/user-management/user-management.tsx.ejs

```diff
@@ -137,7 +137,7 @@ export class UserManagement extends React.Component<IUserManagementProps, IPagin
                       </Button>
                     ) : (
                       <Button color="danger" onClick={this.toggleActive(user) }>
-                        Deactivated
+                        Desactivated
```

Comment:
@chrisdns what is this? It should be Deactivated

Reply:
Sorry for this my french got the upper hand...